source/writing/guidelines.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Guidelines
+type: page
+nested: true
+parent: Writing
+---
+
+# Guidelines
+
+#### Be concise
+- Use as few words as possible. Avoid unnecessary and redundant information.
+- Avoid large blocks of text.
+- Avoid long, complex sentences.
+
+#### Be conversational
+- Use natural, conversational language, no bullshit.
+- Contractions are okay to be used. "Here's a list of projects we’ve released & contributed to." Instead of "Here is a list of projects we have released & contributed to". They make the text more human
+- Avoid developer/designer-focused terminology, unless you’re writing for them.
+
+#### Be direct
+- Use active voice "The boy hit the ball" not "The ball was hit by the boy".
+- Avoid complex verb structures.
+
+#### Be Positive
+-  Remember, we empower positive change.
+- Always try to write sentences positively, not negatively.
+
+# Basic how-to
+
+If you don't read anything else just read this:
+
+- When you are writing here you are representing Startaê. You are speaking for yourself and for it
+- Think problem are you solving to who is reading and how are you solving it
```

Comment:
This sentence is unclear to me. I'm not sure if I got, but how about something like:

> Think about the problem you are solving in a perspective of the reader, and how are you solving it